warhorn/warhorn.py

```diff
@@ -400,7 +379,16 @@ def clone_warrior_and_tools(base_path, current_dir, repo_root, **kwargs):
                                               console_log_name=console_log_name,
                                               print_log_name=print_log_name)
                 getDone()
-
+    overwrite = kwargs.get("overwrite")
+    dest_dir = os.path.join(base_path, "warrior/Tools/"+repo_name)
```

Comment:
I think join functions accept more than 2 arguments, we can separate warrior Tools and repo_name into different parts

Reply:
Done. Please check and approve.